WordPress/src/main/java/org/wordpress/android/ui/stories/StoryComposerViewModel.kt

```diff
@@ -52,6 +55,9 @@ class StoryComposerViewModel @Inject constructor(
     private val _submitButtonClicked = MutableLiveData<Event<Unit>>()
     val submitButtonClicked: LiveData<Event<Unit>> = _submitButtonClicked
 
+    private val _shouldShowOptimizeImagesAdvertising = MutableLiveData<Event<() -> Unit>>()
```

Comment:
We might want to consider renaming this to something like `advertizeImageOptimizationEvent` or `showOptimizeImagesDialog` or similar. `Should show` evokes "boolean" result to me, wdyt?

Reply:
👍 addressed in 2b583f9ee3